src/main/java/org/infinispan/persistence/redis/RedisStore.java

```diff
@@ -37,7 +37,7 @@
     @Override
     public void init(InitializationContext ctx)
     {
-        RedisStore.log.info("Redis cache store initialising");
+        RedisStore.log.info("Initialising Redis store for cache " + ctx.getCache().getName());
```

Comment:
Better to use:
`RedisStore.log.info("Initialising Redis store for cache '%s' ", ctx.getCache().getName());`